test/query.spec.js

```diff
@@ -90,16 +90,16 @@ describe('Query', () => {
 
     let i = 0
     const visited = []
-    const query = (p, cb) => {
+    const query = async (p) => {
```

Comment:
same as before, and other occurrences below that I will not mark in this review

Reply:
see response above